### ISSUE TITLE
excluded routes from NYC subway

### ISSUE THREAD

**Issue by anastasiaclark** [anastasiaclark/MTA-Mass-Transit#5]
route_id | route_short | route_long | color
-- | -- | -- | -- 
5X | 5X | Lexington Avenue Express | 00933C
6X | 6X | Lexington Avenue Express | 00A65C
7X | 7X | Flushing Express | B933AD
Z | Z | Nassau St Express | 996633

@frankpd there are some express routes that don't get written out in NYC Subway Routes. Do we want to keep excluding them or should we keep 6X, 7X and Z (segments of 5X are intentionally dropped as unusual service)?

**Comment by frankpd** [comment]
@anastasiaclark the "X" services are special rush hour versions of the regular services that skip many stops, but follow the same route as the regular service. We dropped them because we didn't want to represent them as distinct routes. The Z train is a special rush hour service that follows the same route as the J train. Rather than representing it as an individual line, we dropped it and treat the J route as the JZ route. In both these cases, we're essentially representing these routes the way they appear on the official MTA subway map.

Sorry for my delay in replying, I just got back from vacation :)
